drinks_touch/config.py

```diff
@@ -8,6 +8,14 @@
 LDAP_USER = os.environ.get("LDAP_USER", "cn=admin,dc=flipdot,dc=org")
 LDAP_PW = os.environ.get("LDAP_PW", "admin")
 
+OIDC_DISCOVERY_URL = os.environ.get("OIDC_DISCOVERY_URL", "http://localhost:8080/realms/flipdot/.well-known/openid-configuration")
+OIDC_CLIENT_ID = os.environ.get("OIDC_CLIENT_ID", "drinks-touch")
+# Client secret default from the shipped realm.json
+OIDC_CLIENT_SECRET = os.environ.get("OIDC_CLIENT_SECRET", "ozlsD9WZi8PfsnYwPO3UFoIUBjPyS4Bd")
```

Comment:
Reading a secret value in a PR is always emotionally challenging xD

Reply:
Sure, but the secret just applies to the local keycloak, so I don't see a risk, while at the same time it makes dev setup way more comfortable. Any suggestion for another solution to achieve this while mitigating some risk?